Kitodo/src/main/java/de/sub/goobi/forms/ProzessverwaltungForm.java

```diff
@@ -708,8 +708,17 @@ public String WerkstueckEigenschaftUebernehmen() {
     public String newTask() {
         this.task = new Task();
         this.task.setProcess(this.process);
+        this.process.getTasks().add(this.task);
+        try {
+            serviceManager.getTaskService().save(task);
+        } catch (DataException e) {
+            logger.error(e);
+        }
+        save();
         this.modusBearbeiten = "schritt";
-        return "/pages/inc_Prozessverwaltung/schritt";
+        this.taskId = this.task.getId();
+        //TODO: enforce http://localhost:8080/kitodo/pages/inc_Prozessverwaltung/schritt.jsf?id=6
```

Comment:
I don't think that the content of TODO is correct as only developer are running Kitodo.Production on localhost.